apps/codeforafrica/src/pages/admin/index.page.js

```diff
@@ -17,6 +19,8 @@ function AdminPage(props) {
       </Head>
       <NextSeo {...props} />
       <Script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js" />
+      <Script src="https://unpkg.com/netlify-cms-widget-uuid-v4@^1.0.12/dist/index.js" />
+      <CustomWidget />
```

Comment:
I think lets use normal async script loading for now so that we're sure things are loaded before `load` event is fired.
```jsx
/* eslint-env browser */
import { NextSeo } from "next-seo";
import Head from "next/head";
import React from "react";

function AdminPage(props) {
  return (
    <>
      <Head>
        {/* Note the "type" and "rel" attribute values, which are required. */}
        <link
          href="/api/admin/config.yml"
          type="text/yaml"
          // eslint-disable-next-line react/no-invalid-html-attribute
          rel="cms-config-url"
        />
        <script
          src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"
          async
        />
        <script
          src="https://unpkg.com/netlify-cms-widget-uuid-v4@^1.0.12/dist/index.js"
          async
        />
        <script
          // eslint-disable-next-line react/no-danger
          dangerouslySetInnerHTML={{
            __html: `
              window.addEventListener("load", () => {
                window.CMS.registerWidget(
                  "uuid",
                  window.uuidWidget.UuidControl
                );
              });
            `,
          }}
        />
      </Head>
      <NextSeo {...props} />
    </>
  );
}

export default AdminPage;

```


... And I'm skipping registering Preview because the UUID is not part of content & hence shouldn't be visible to the users.